### ISSUE TITLE
Duplicate compile error message in ionide 4

### ISSUE THREAD

**Issue by forki** [ionide/ionide-vscode-fsharp#1122]
![image](https://user-images.githubusercontent.com/57396/59336130-3fb63b80-8cfe-11e9-856e-e158cac526af.png)

**Comment by Krzysztof-Cieslak** [comment]
Is that something reproducible? I've never seen that

**Comment by forki** [comment]
let me check. It may be related to the out-of-date cache issue

**Comment by forki** [comment]
ok can't repro. will reopen when I can

**Comment by forki** [comment]
![image](https://user-images.githubusercontent.com/57396/59408685-e0673280-8db4-11e9-852c-69d6c3673757.png)

it's still there.

```
let xs =
    let ways =
        [||]        
        |> Set.ofArray // distinct and sorted
        |> Seq.groupBy fst
    
    ways
    |> Array.map (fun (way,runs) ->
        
    )

```

**Comment by Krzysztof-Cieslak** [comment]
Can repro 😕

**Comment by forki** [comment]
can or can't?

**Comment by Krzysztof-Cieslak** [comment]
Can

**Comment by Krzysztof-Cieslak** [comment]
Interesting.... it seems I get duplicated errors from FCS O_o

**Comment by forki** [comment]
Yes maybe we run into fcs bug

**Comment by Krzysztof-Cieslak** [comment]
I'll just `Array.distinct` it, someone else can worry about FCS ;-)

**Comment by forki** [comment]
I'm just worried it does twice the work. /cc @cartermp @dsyme

**Comment by Krzysztof-Cieslak** [comment]
Workaround by https://github.com/fsharp/FsAutoComplete/commit/ee95c5c69e34459f420ab2b606326d5806ec1a9b

**Comment by Krzysztof-Cieslak** [comment]
Fix released in experimental `0.3.2`
